sdk/tests/invariant.test.ts

```diff
@@ -0,0 +1,246 @@
+import type { Liquidity, Percentage } from 'invariant-cspr-wasm'
+import { ALICE, LOCAL_NODE_URL } from '../src/consts'
+import { Key, Network } from '../src/enums'
+import { Erc20 } from '../src/erc20'
+import { Invariant } from '../src/invariant'
+import { assertThrowsAsync, deployInvariantAndTokens, loadChai } from '../src/testUtils'
+import { callWasm, initCasperClient, loadWasm } from '../src/utils'
+
+let hashes: {
+  invariant: { loadHash: string; packageHash: string }
+  tokenX: { loadHash: string; packageHash: string }
+  tokenY: { loadHash: string; packageHash: string }
+}
+
+describe('invariant test', () => {
+  const client = initCasperClient(LOCAL_NODE_URL)
+  const deployer = ALICE
+  const network = Network.Local
+
+  beforeEach(async () => {
+    hashes = await deployInvariantAndTokens(client, deployer)
+  })
+
+  it('should change protocol fee', async () => {
+    const chai = await loadChai()
+    const invariant = await Invariant.load(client, hashes.invariant.loadHash, network)
+    const newFee: Percentage = { v: 20000000000n }
+    await invariant.changeProtocolFee(deployer, newFee)
+
+    const { protocolFee } = await invariant.getInvariantConfig()
+    chai.assert.deepEqual(protocolFee, newFee)
+  })
+
+  it('should add fee tier', async () => {
+    const wasm = await loadWasm()
+    const chai = await loadChai()
+    const invariant = await Invariant.load(client, hashes.invariant.loadHash, network)
+    const feeTier = await callWasm(wasm.newFeeTier, { v: 6000000000n }, 10n)
```

Comment:
Wouldn't it be better to hide such `wasm` invocations on the sdk client side, especially considering their usage from the frontend perspective?
```Typescript
createFeeTier(fee: Percentage, tickSpacing: bigint): FeeTier
```